rcp/org.testeditor.web.testexecutionlogview/login_test/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
I created a separate ticket for [TE-481|that https://testeditor.atlassian.net/browse/TE-481]